lib/ConnectionContextBase.ts

```diff
@@ -7,6 +7,8 @@ import { DataTransformer, DefaultDataTransformer } from "./dataTransformer";
 import { TokenProvider } from "./auth/token";
 import { ConnectionConfig } from "./connectionConfig";
 import { SasTokenProvider } from "./auth/sas";
+import rhea from "rhea";
```

Comment:
As discussed offline, lets log an issue in rhea-promise to export `websocket_connect` and use it from there

Reply:
Can this be done in a separate PR or is that change required at this point?

Reply:
@AlexGhiondea separate PR, not blocking.